src/app/frontend/chrome/notifications/component.ts

```diff
@@ -20,7 +20,7 @@ import {Notification, NotificationsService} from '../../common/services/global/n
 @Component({
   selector: 'kd-notifications',
   templateUrl: './template.html',
-  styleUrls: ['./style.scss'],
+  styleUrls: ['./style.scss', './rtl.scss'],
```

Comment:
Why have you added another file?

Reply:
To keep the main style file clean and keep RTL override rules on another file.
I can merge them together if you think it's better.